experimental/examples/bubblebench/common/src/rnd.ts

```diff
@@ -5,16 +5,15 @@
 
 import { Random } from "best-random";
 
-// eslint-disable-next-line no-bitwise
-export const rnd = new Random((Math.random() * 0x100000000) | 0);
+export const rnd = new Random(Math.trunc(Math.random() * 0x1_00_00_00_00));
 
-export function randomColor() {
-    // eslint-disable-next-line no-bitwise
-    const channel = () => (32 + (rnd.float64() * 196) | 0).toString(16).padStart(2, "0");
```

Comment:
Since bubblebench is used for benchmarking, probably worth having @seanimam confirm that this potential slowdown (`Math.trunc()` instead of `| 0`) isn't a deal breaker.

Reply:
For context, the rule being enforced here is: https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/prefer-math-trunc.md

Reply:
If we think this is perf critical, I am happy to disable the rule here with a comment explaining why.

Reply:
I went ahead and just disabled the rule here for now. This can be cleaned up separately if we wish.